pkg/analytics/analytics.go

```diff
@@ -0,0 +1,73 @@
+/*
+Copyright 2019 LitmusChaos Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package analytics
+
+import (
+	"crypto/rand"
+	"fmt"
+
+	ga "github.com/jpillora/go-ogle-analytics"
+)
+
+// UUIDGenerator creates a new UUID each time a new user triggers an event
+func UUIDGenerator() (string, error) {
+	b := make([]byte, 16)
+	_, err := rand.Read(b)
+	if err != nil {
+		return "", err
+	}
+	uuid := fmt.Sprintf("%x-%x-%x-%x-%x", b[0:4], b[4:6], b[6:8], b[8:10], b[10:])
+	return uuid, err
+}
+
+// TriggerAnalytics is reponsible for sending out events
+func TriggerAnalytics() (string, error) {
+
```

Comment:
You don't have to return `string` from this function

Reply:
Yeah, fixed in accordance to aforementioned comment.